src/components/dropdowns/Dropdown.spec.jsx

```diff
@@ -63,4 +63,23 @@ describe('<Dropdown />', () => {
     component.update();
     expect(component.find('.sg-dropdown__items')).toHaveLength(0);
   });
+
+  it('should pass event and link to onSelectHanlder', () => {
+    const onItemSelect = jest.fn();
+    const component = mount(
```

Comment:
Do you need `mount` instead of `shallow`?

Reply:
will check

Reply:
The only small disadvantage is, it will not test if the function received anything as the first argument (`event` object in this case), and we can only assert on the second one. (as we are missing event system in shallow, it will receive `undefined`)
If you are ok with that I can switch to `shallow` :)

Reply:
changed in 7a6cfdac

Reply:
Thanks, looks good